visualization/app/codeCharta/ui/settingsPanel/settingsPanel.po.ts

```diff
@@ -1,10 +1,25 @@
+import { delay } from "../../../puppeteer.helper"
+
 export class SettingsPanelPageObject {
+	constructor(private page) {}
 
-    constructor(private page) {}
+	public async open() {
+		await this.page.click("#actionButtons > sidenav-toggle-component > button")
+		await delay(500)
+	}
 
-    public async toggleMapsPanel() {
-        const selector = 'settings-panel-component > md-expansion-panel-group > md-expansion-panel';
-        return this.page.click(selector);
-    }
+	public async toggleTreeViewSearchPanel() {
+		await this.page.click(
+			"settings-panel-component > md-expansion-panel-group > md-expansion-panel:nth-child(1) > md-expansion-panel-collapsed"
+		)
+		await delay(500)
+	}
 
-}
\ No newline at end of file
+	public async rightClickRootNodeInTreeViewSearchPanel() {
+		await this.page.click(
+			"#codeMap > sidenav-component > md-sidenav > md-content > settings-panel-component > md-expansion-panel-group > md-expansion-panel:nth-child(1) > md-expansion-panel-expanded > md-expansion-panel-content > map-tree-view-component > map-tree-view-level-component > div > div.tree-element-label-0",
```

Comment:
Can you think of another option than using `:nth-child(1)` to select the correct `md-expansion-panel`? Using an `id`-tag only for the test is probably not better..

Reply:
something like this should work sidenav-component map-tree-view-level-component > div > div.tree-element-label-0

Reply:
md-expansion-panel md-expansion-panel-collapsed